remove mention of building blocks and blocks from documentation (#1660)
open-contracting/standard: docs/guidance/map/amendments.md

* **The release tag** (`tag`). For example, for a release with a new contract, use 'contract'. For an update to the contract, use 'contractUpdate', and for an amendment to the contract, use 'contractAmendment'.

> * **The amendments** object. This can contain an array of amendment explanations, and clearly identify the releases that contain before and after values.

Reviewer: `amendments` is an array.

```suggestion
* **The amendments** array. Each item in the array is an `Amendment` object, including a rationale, a description, and references to the releases that contain before and after values.
```

#### Tender Amendment

> A few days later, the publisher increases the value of the tender and extends the deadline for bid submissions. These changes are considered as an 'amendment' by the publisher (depending on jurisdiction, certain changes can need to be disclosed as amendments), and so the new release has the `tag` 'tenderAmendment' and an `amendments` object under `tender`. The release reflects the updated value (USD 2000 instead of USD 1000) and the updated closing date for bid submissions (`2012-02-20` instead of `2012-02-15`). See the JSON example below.

Reviewer: ```suggestion
A few days later, the publisher increases the value of the tender and extends the deadline for bid submissions. These changes are considered as an 'amendment' by the publisher (depending on jurisdiction, certain changes can need to be disclosed as amendments), and so the new release has the `tag` 'tenderAmendment' and an `amendments` array under `tender`. The release reflects the updated value (USD 2000 instead of USD 1000) and the updated closing date for bid submissions (`2012-02-20` instead of `2012-02-15`). See the JSON example below.
```

Note in this example that:

> * **The amendments object does not contain data on what was changed**. Changes are recorded by updating the fields of the `tender` object a new release.

Reviewer: ```suggestion
* **The amendments array does not contain data on what was changed**. Changes are recorded by updating the fields of the `tender` object in a new release.
```

#### Contract Amendment

> A few days after the contract release, its scope is increased to include the purchase of one additional appliance. A new 'contractAmendment' release is built, where a single item is added in the `contracts/items` object and the value of the contract is increased. A `amendments` object is included to explain the rationale of the changes.

Reviewer: ```suggestion
A few days after the contract release, its scope is increased to include the purchase of one additional appliance. A new 'contractAmendment' release is built, where a single item is added in the `contracts/items` array and the value of the contract is increased. A `amendments` array is included to explain the rationale of the changes.
```
